packages/horizontal-slider-react/src/HorizontalSlider.tsx

```diff
@@ -0,0 +1,183 @@
+  const sliderIdRef = useRef(generateHorizontalSliderId());
+
+  function scrollLeft() {
+    if (listRef.current) {
+      listRef.current.scrollLeft -= listRef.current.offsetWidth * 0.75;
+    }
+  }
+
+  function scrollRight() {
+    if (listRef.current) {
+      listRef.current.scrollLeft += listRef.current.offsetWidth * 0.75;
+    }
+  }
+
+  function updateNavigationButtonsProps() {
+    if (
+      !listRef.current ||
+      !previousButtonRef.current ||
+      !nextButtonRef.current
+    ) {
+      return;
+    }
+
+    previousButtonRef.current.hidden = listRef.current.scrollLeft <= 0;
+    nextButtonRef.current.hidden =
+      listRef.current.scrollLeft + listRef.current.clientWidth >=
+      listRef.current.scrollWidth;
+  }
+
+  useEffect(() => {
+    updateNavigationButtonsProps();
+  });
+
+  if (props.items.length === 0) {
+    return null;
+  }
+
+  return (
+    <div
+      className={cx('uic-HorizontalSlider-container', props.classNames?.root)}
+    >
+      <button
+        ref={previousButtonRef}
+        title={translations.previousButtonTitle}
+        aria-label={translations.previousButtonLabel}
+        aria-controls={sliderIdRef.current}
+        className={cx(
+          'uic-HorizontalSlider-navigation',
+          'uic-HorizontalSlider-navigation--previous',
+          props.classNames?.navigation,
+          props.classNames?.navigationPrevious
+        )}
+        onClick={(event) => {
+          event.preventDefault();
+          scrollLeft();
+        }}
+      >
+        <svg width="8" height="16" viewBox="0 0 8 16" fill="none">
+          <path
+            fillRule="evenodd"
+            clipRule="evenodd"
+            d="M7.13809 0.744078C7.39844 1.06951 7.39844 1.59715 7.13809 1.92259L2.27616 8L7.13809 14.0774C7.39844 14.4028 7.39844 14.9305 7.13809 15.2559C6.87774 15.5814 6.45563 15.5814 6.19528 15.2559L0.861949 8.58926C0.6016 8.26382 0.6016 7.73618 0.861949 7.41074L6.19528 0.744078C6.45563 0.418641 6.87774 0.418641 7.13809 0.744078Z"
+            fill="currentColor"
+          />
+        </svg>
+      </button>
+
+      <ol
+        className={cx('uic-HorizontalSlider-list', props.classNames?.list)}
+        ref={listRef}
+        tabIndex={0}
+        id={sliderIdRef.current}
+        aria-roledescription="carousel"
+        aria-label={translations.sliderLabel}
+        aria-live="polite"
+        onScroll={updateNavigationButtonsProps}
+        onKeyDown={(event) => {
+          if (event.key === 'ArrowLeft') {
+            event.preventDefault();
+            scrollLeft();
+          } else if (event.key === 'ArrowRight') {
+            event.preventDefault();
+            scrollRight();
+          }
+        }}
+      >
+        {props.items.map((item, index) => (
+          <li
+            key={item.objectID}
+            className={cx('uic-HorizontalSlider-item', props.classNames?.item)}
+            aria-roledescription="slide"
+            aria-label={`${index + 1} of ${props.items.length}`}
```

Comment:
We might want to provide this word as a translations, wdyt? 

Maybe something like `sliderItemLabel` and provide a small caption with an example, defaulted to `of`

Reply:
if we have this as a translation, it likely would need to be completely a translation, since the order won't be the same in every language, and we didn't accept functions as translations. Not sure what's the solution

Reply:
That's true but it would cover most cases until we encounter this issue, I guess it's better than having English by default

Reply:
An other/complementary solution for this could be to provide a kind-of RTF boolean variable that would swap the places of the two numbers if setted

Reply:
That's tricky because that should be a function indeed.

We can still introduce this is a function translation later because we need to release today. Wdyt?

Reply:
Ooookay